core/src/main/scala/com/microsoft/azure/synapse/ml/fabric/FabricTokenParser.scala

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.fabric
+
+import spray.json._
+
+class InvalidJwtTokenException(message: String) extends Exception(message)
+
+class JwtTokenExpiryMissingException(message: String) extends Exception(message)
+
+class FabricTokenParser(JWToken: String) {
```

Comment:
make all the APIs private or private[ml] if they arent supposed to be used by a end user